main.go

```diff
@@ -252,6 +254,9 @@ func main() {
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
 
+	log.Base().SetLevel(*logLevel)
```

Comment:
Yes, but that would be linking a logging library to a CLI/Flag library. Having a function, and then duplicating it in every repo is cleaner.